src/kOS.Safe/Compilation/KS/Compiler.cs

```diff
@@ -685,8 +685,8 @@ private void PreProcessUserFunctionStatement(ParseNode node)
 
                 // build default dummy function to be used when this is a LOCK:
                 currentCodeSection = userFuncObject.GetUserFunctionOpcodes(0);
-                AddOpcode(new OpcodeArgBottom());
-                AddOpcode(new OpcodePush("$" + userFuncObject.ScopelessIdentifier)).Label = userFuncObject.DefaultLabel;
+                AddOpcode(new OpcodeArgBottom()).Label = userFuncObject.DefaultLabel;;
```

Comment:
Won't this cause the ReplaceLabels method in ProgramBuilder to change the default label to point at "ArgBottom" opcode?